lib/api.js

```diff
@@ -6,7 +6,8 @@ function Api (app, requireName) {
 }
 
 Api.prototype.initialize = function () {
-  return this.load().then(this.addApiCommands.bind(this))
+  var self = this
+  return self.load().then(self.addApiCommands.bind(self))
```

Comment:
no need to define `self`
```suggestion
  return this.load().then(this.addApiCommands.bind(this))
```